napari-workshops/installation.md

```diff
@@ -98,53 +98,43 @@ as well and you can skip to the next section.
 2. We use an environment to encapsulate the Python tools used for this workshop.
    This ensures that the requirements for this workshop do not interfere with
    your other Python projects. To create the environment (named
-   `napari-tutorial`) and install Python 3.9 in it, enter the following command:
+   `napari-workshop`) and install Python 3.11 and napari in it, enter the following command:
 
     ```bash
-    conda create -n napari-tutorial python=3.9
+    conda create -n napari-workshop -c conda-forge python=3.11 napari pyqt jupyterlab
     ```
 
 3. Once the environment setup has finished, activate the environment:
 
     ```bash
-    conda activate napari-tutorial
+    conda activate napari-workshop
     ```
 
     If you successfully activated the environment, you should now see
-   `(napari-tutorial)` to the left of your command prompt.
+   `(napari-workshop)` to the left of your command prompt.
 
-4. Install the workshop dependencies with the commands below.
+4. Install the additional workshop dependencies with the commands below.  
 
-    **If you're on an M1 Mac**:
+   For the plugin template:
 
     ```bash
-    conda install -c conda-forge notebook napari
-    python -m pip install cookiecutter magicgui
-    python -m pip install stardist-napari
+    conda install -c conda-forge cookiecutter 
```

Comment:
napari pulls in magicgui, so we don't need that as it's own thing and it's best to just stick to conda for everything IMO.